src/statsig/constants.ts

```diff
@@ -37,6 +37,7 @@ export const FeatureGates = {
   [StatsigFeatureGates.SHOW_MULTI_CHAIN_TRANSFERS]: false,
   [StatsigFeatureGates.SHOW_NATIVE_TOKENS]: false,
   [StatsigFeatureGates.SHOW_ETH_IN_CICO]: false,
+  [StatsigFeatureGates.FETCH_MULTI_CHAIN_BALANCES]: false,
```

Comment:
I have some concerns about the naming of this, and possibly the usage of it; I haven't gotten that far yet.

I'd guess that we'll want to continue to add chains, rather than revert to a single chain some time after shipping Ethereum. If that's the case, there will be some cleanup after shipping at 100%. Likely we wouldn't get to this cleanup until we're trying to ship an additional chain, at which point we would replace the set of feature gates for the chain that was rolled out with the next chain we want to ship. 

This gate is named as if we'll continue to add chains with a single feature gate, maintaining the option to revert back to a single chain. I just don't see that happening.

Reply:
same concern applies to the show_multi_chain_transfers feature flag

Reply:
I have a separate, less significant concern about the "fetch" part of the feature gate name-- almost makes it sounds like there will be some other feature gate for whether to actually show the multichain balances.

Reply:
Interesting -- this is a good point. So, you're basically saying that, steady state, in production, we'll support some number of chains, while more chains could be "in development" s.t. we want access to them on dev builds/wallets, but don't necessarily want them to be available in production?

That use case above is _kind of_ what I had in mind with the `supportedChains` concept from an earlier PR; e.g., the wallet maintains a "full" list of chains that it knows about, some _subset_ of which are available in production. Overriding a feature gate in Statsig would allow the user to see the "full" list of chains in-app. Both the full list of chains, and the subset of "prod-available" chains, could be configured remotely by a dynamic config in Statsig. I think this would also mean we don't need new feature gates for each chain we want to roll out; we'd just add it to the prod-available subset when we're ready. (Actually, we could probably do this with a single dynamic config that contains prod-available chains, and specify overrides/targeting rules to return an expanded list with chains that are in development -- thinking out loud here.)

This of course assumes that we don't want finer granularity in multi-chain features, e.g., we wouldn't be able to turn on/off token balance or transfer feed support for an arbitrary chain. I guess we could follow the above pattern per-feature, if we wanted to.

Reply:
Discussed this offline; a more comprehensive solution will come in a followup PR; will keep this as-is for now.